### ISSUE TITLE
Menu stops scrolling in prop emotes 

### ISSUE THREAD

**Issue by Ell34** [alberttheprince/rpemotes-reborn#78]
First ticket was closed due to no response, my apologies. Myself and players have noticed an issue when in the prop emotes section. Scrolling is a bit laggy and then all of the sudden you cannot scroll any further. When scrolling from the top to bottom, this occurs on cig, cigar and cigar2 (sometimes joint). Going from the bottom to top it occurs on Steal Tire 2.

**Comment by alberttheprince** [comment]
can you paste the config for these emotes in specific? We'll compare with what we have and see if there's any differences.

**Comment by Ell34** [comment]
Can do. Just give me a bit as the server owner is away on holidays.

**Comment by alberttheprince** [comment]
Were you able to resolve this?

**Comment by Ell34** [comment]
No, server dev is swamped and this isn't on his priority list :(

**Comment by iSentrie** [comment]
it supposed to slow down on these, but i dont believe it breaks, once animation finish, it should continue working

**Comment by alberttheprince** [comment]
Going to close this for now, as we don't have more details to solve/research this. Please reopen with video/info of the issue when you can.
